### ISSUE TITLE
Daily - Jueves 23/10/2025

### ISSUE THREAD

**Issue by nikaLPFB** [BreakfastProject/proyecto7_ensemble_grupo2#21]
# Dailies Standup - Jueves 23/10/2025

## Asistentes
@MariaDunaeva1  @LadyFB89  @Yedpt  @CristianYepes 

### @MariaDunaeva1 
- [ ] Implementacion de CNN y del Modelo LightGBM
- [ ] Mejorar el resultado de las metricas de clasificacion( accuracy, recall, precision, F1 score, curva ROC, 
   matrices de confusión, etc.)

### @Yedpt 
- [ ] Implementacion de  CNN y del  Modelo XGBoost 
- [ ] Mejorar el resultado de las metricas de clasificacion( accuracy, recall, precision, F1 score, curva ROC, 
   matrices de confusión, etc.)

### @nikaLPFB 
- [ ] Implementacion de CNN  y luego Random Forest
- [ ] Mejorar el resultado de las metricas de clasificacion( accuracy, recall, precision, F1 score, curva ROC, 
   matrices de confusión, etc.)
- [ ] Realizar el Dailies diario

###  @CristianYepes 
- [ ]  Implementacion de CNN y Fast API
- [ ] Mejorar el resultado de las metricas de clasificacion( accuracy, recall, precision, F1 score, curva ROC, 
   matrices de confusión, etc.)

## Observaciones
Cristian , se conecta por las tardes razones  de estudio

Jueves 23/10/2025
Asignado a: Lady Flores Berrocal
Etiquetas: daily, standup
Estado: Abierto
